app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessUpdate.kt

```diff
@@ -2,9 +2,12 @@ package org.simple.clinic.teleconsultlog.success
 
 import com.spotify.mobius.Next
 import com.spotify.mobius.Update
+import org.simple.clinic.mobius.next
 
 class TeleConsultSuccessUpdate : Update<TeleConsultSuccessModel, TeleConsultSuccessEvent, TeleConsultSuccessEffect> {
   override fun update(model: TeleConsultSuccessModel, event: TeleConsultSuccessEvent): Next<TeleConsultSuccessModel, TeleConsultSuccessEffect> {
-    return Next.noChange()
+    return when (event) {
+      is PatientDetailsLoaded -> next(model.patientDetailLoaded(event.patient))
```

Comment:
In a previous commit, we load the patient details whenever the screen is created. In the screen restoration scenario, if the model already has the patient details loaded we do not need to load it again, right?

Reply:
The current change makes it so that the patient details are never loaded since there is no effect being emitted at all. We should emit the effect conditionally based on whether the model has the details or not.

Reply:
Please ignore the earlier comment, I was only looking a single commit instead of the full set of changes.